includes/admin/class-wc-admin-addons.php

```diff
@@ -112,7 +112,22 @@ public static function get_section_data( $section_id ) {
 			}
 		}
 
-		return apply_filters( 'woocommerce_addons_section_data', $section_data->products, $section_id );
+		return array(
+				"blocks" => ! empty( $section_data->blocks )
+					? apply_filters(
+						'woocommerce_addons_section_blocks_data',
+						$section_data->blocks,
+						$section_id
+					)
+					: array(),
+				"products" => ! empty(  $section_data->products )
+					? apply_filters(
+						'woocommerce_addons_section_data',
+						$section_data->products,
+						$section_id
+					)
+					: array(),
```

Comment:
The filters here should always be applied. If the `$section_data` properties need to be preprocessed (and ensured to as least be empty arrays), then so be it. Please also use single quotes for the array keys. ([#](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#single-and-double-quotes))

Sidenote: always test if a statement is true in ternary operators. ([#](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#ternary-operator))